src/renderers/dom/client/__tests__/ReactMount-test.js

```diff
@@ -169,25 +169,27 @@ describe('ReactMount', function() {
     );
   });
 
-  (WebComponents === undefined ? xit : it)
```

Comment:
There's whitespace between the function identifier and the open-paren designating the argument list.